packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -1244,6 +1250,20 @@ export function attach(
       recordMount(fiber, parentFiber);
     }
 
+    if (traceUpdatesEnabled) {
+      const elementType = getElementTypeForFiber(fiber);
```

Comment:
You can move this inside the `if (traceNearestHostComponentUpdate)` condition now that you only use it in one of  the branches.